### PR TITLE
Test math logic

### PR DESCRIPTION
**Pull request by gdumani**
This pull request includes tests for the following components:
|Component |Test Performed |
|--------|--------|
| App | Renders correctly |
| Home | Renders correctly |
| Header | Renders correctly |
| Calculate | Logic works correctly |
| Operate | Logic works correctly |
| Calculator | Renders correctly and UI works correctly |
| Fetch | Renders correctly |

### CODE REVIEW

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    const myObj = { total: '2', next: '2', operation: '÷' };
+    const ResObj = { total: '0', next: null, operation: null };
+    expect(calculate(myObj, 'AC')).toEqual(ResObj);
+  });
+  test('will return the value of 4 by multiply 3 x 2', () => {
+    const myObj = { total: '3', next: '2', operation: 'x' };
+    const ResObj = { total: '6', next: null, operation: null };
+    expect(calculate(myObj, '=')).toEqual(ResObj);
+  });
+});
```
Comment (tresorsawasawa) [inline src/__tests__/calculate.test.js]: - [x] As required for the project all the functionalities & scenarios of the `calculate.js` module to be covered using Jest, kindly make sure you add tests scenario to `'+/-'`,  `decimal numbers` (i.e. numbers including `.` e.g. `3.5`, `4.23`,...), etc.

File: src/__tests__/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  test('shoud return 4 after adding 7 - 3', () => {
+    expect(operate('7', '3', '-')).toEqual('4');
+  });
+  test('shoud return 21 after adding 7 x 3', () => {
+    expect(operate('7', '3', 'x')).toEqual('21');
+  });
+  test('shoud return 4 after adding 12 ÷ 3', () => {
+    expect(operate('12', '3', '÷')).toEqual('4');
+  });
+});
```
Comment (tresorsawasawa) [inline src/__tests__/operate.test.js]: - [x] As required for the project all the functionalities & scenarios of the `operate.js` module to be covered using Jest, kindly make sure you add tests scenario to `%`, `raises an error when dividing number by 0`,....

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @gdumani and @mwismann 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 
- All tests are passing :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @gdumani and @mwismann, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

No linter errors ✔️
Descriptive PR ✔️
Correct Git flow ✔️
Professional ReADME file ✔️
All tests passed 💯

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**